### DISCUSSION TITLE
Custom Header and Select Inputs in Mantine DatePicker

### DISCUSSION THREAD

**Discussion by IvanNavin** [mantinedev/mantine#4663] [Q&A]
Hello Mantine Support Team,

I hope this message finds you well. I have been using the Mantine library for my project and I must say that I am quite impressed with its features and performance. However, I have a specific requirement regarding the DatePicker component that I would like to discuss.

In my design, I have a datePicker with two columns, and I would like to have a custom header that includes two select inputs for choosing the month and year. It's crucial that these inputs are select elements. I've seen similar functionality in other date picker libraries, and I am wondering if it's possible to achieve this with your DatePicker component.

For reference, here are links to other libraries where this functionality has been implemented:
[https://reactdatepicker.com/#example-custom-header](https://reactdatepicker.com/#example-custom-header)
[https://react-dates.github.io/react-dates/?path=/story/daypickersingledatecontroller--single-month-custom-caption](https://react-dates.github.io/react-dates/?path=/story/daypickersingledatecontroller--single-month-custom-caption)

I genuinely wish to continue using Mantine for my project, as I have had a positive experience so far. However, the absence of this particular feature is a concern for me. If it's possible to achieve the custom header with select inputs using Mantine's DatePicker, it would greatly enhance the user experience of my application.

I would greatly appreciate it if you could provide me with guidance or a solution to implement this feature using Mantine's DatePicker. Your assistance in this matter would help me make an informed decision about which library to continue using for my project.

Thank you for your time and consideration. I look forward to your response.

Best regards,
Holovko Ivan

**Comment by rtivital** [comment]
1. Please use GitHub Discussions
2. To answer your question:

Hide library controls with CSS, then add your own controls with month/year picker, use controlled `date`
